File: Oophw5/Interfaces/ConsoleView.java
package Oophw5.Interfaces;

public class ConsoleView extends View {
    Menus m = new Menus();

    @Override
    public void menu0() {
        System.out.println(m.menuMain);
    }

    @Override
    public void menuSearch() {
        System.out.println(m.menuSearch);
    }

    @Override
    public void mDeletePerson() {
        System.out.println(m.menuSearch);
    }

    @Override
    public void mAddPerson() {
        System.out.println(m.menuAdd);
    }

    @Override
    public void mAddPhoneToPerson() {
        System.out.println(m.menuSearch);
    }

    @Override
    public void textOut(String string) {
        System.out.println(string);
    }

}

File: oophw7/interfaces/consoleview.py
from interfaces.menus_hw7 import menus
from interfaces.view_hw7 import viewhw7


class consoleview(viewhw7):
    m = menus()

    def menu0(conv):
        print(conv.m.menuMain)

    def menuSearch(conv):
        print(conv.m.menuSearch)
    
    def mAddPhoneToPerson(conv):
         print(conv.m.menuAddPhone)

    def mDeletePerson(conv):
        print(conv.m.menuDelete)

    def mAddPerson(conv):
        print(conv.m.menuAdd)

    def textout(conv, text: str):
        print(text)
